model/span/event_test.go

```diff
@@ -29,15 +29,16 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
+	"github.com/elastic/beats/libbeat/common"
+
 	m "github.com/elastic/apm-server/model"
 	"github.com/elastic/apm-server/model/metadata"
 	"github.com/elastic/apm-server/sourcemap"
 	"github.com/elastic/apm-server/transform"
-	"github.com/elastic/beats/libbeat/common"
 )
 
 func TestDecodeSpan(t *testing.T) {
-	spanTime, _ := time.Parse(time.RFC3339, "2018-05-30T19:53:17.134Z")
+	spanTime := time.Date(2018, 5, 30, 19, 53, 17, 134*1e6, time.UTC)
```

Comment:
honestly, i find L40 more readable

Reply:
I was equal on both but am won over by the compile time failure over runtime/ignoring the failure, even if it is test code.